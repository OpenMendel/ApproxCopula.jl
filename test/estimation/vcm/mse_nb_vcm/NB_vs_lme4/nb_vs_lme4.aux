\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\gdef \LT@i {\LT@entry 
    {1}{51.47208pt}\LT@entry 
    {2}{39.77777pt}\LT@entry 
    {2}{71.44463pt}\LT@entry 
    {2}{71.44463pt}\LT@entry 
    {1}{75.86008pt}}
\gdef \LT@ii {\LT@entry 
    {1}{58.22185pt}\LT@entry 
    {1}{64.30553pt}\LT@entry 
    {2}{58.66687pt}\LT@entry 
    {2}{78.66687pt}\LT@entry 
    {1}{75.86008pt}}
\@writefile{toc}{\contentsline {subsection}{Comparing Parameter Estimates}{1}{section*.1}\protected@file@percent }
\newlabel{comparing-parameter-estimates}{{}{1}{Comparing Parameter Estimates}{section*.1}{}}
\@writefile{toc}{\contentsline {subsection}{Comparing Fit Times (CI times) in Seconds}{1}{section*.2}\protected@file@percent }
\newlabel{comparing-fit-times-ci-times-in-seconds}{{}{1}{Comparing Fit Times (CI times) in Seconds}{section*.2}{}}
\@writefile{toc}{\contentsline {section}{lme4: glmer.nb}{2}{section*.3}\protected@file@percent }
\newlabel{lme4-glmer.nb}{{}{2}{lme4: glmer.nb}{section*.3}{}}
\@writefile{toc}{\contentsline {subsubsection}{Show estimated r from lme4: glmer.nb}{3}{section*.4}\protected@file@percent }
\newlabel{show-estimated-r-from-lme4-glmer.nb}{{}{3}{Show estimated r from lme4: glmer.nb}{section*.4}{}}
\@writefile{toc}{\contentsline {subsubsection}{Show confidence intervals from lme4: glmer.nb}{3}{section*.5}\protected@file@percent }
\newlabel{show-confidence-intervals-from-lme4-glmer.nb}{{}{3}{Show confidence intervals from lme4: glmer.nb}{section*.5}{}}
\@writefile{toc}{\contentsline {section}{glmmTMB: glmmTMB}{4}{section*.6}\protected@file@percent }
\newlabel{glmmtmb-glmmtmb}{{}{4}{glmmTMB: glmmTMB}{section*.6}{}}
\@writefile{toc}{\contentsline {subsubsection}{Show estimated r from glmmTMB}{4}{section*.7}\protected@file@percent }
\newlabel{show-estimated-r-from-glmmtmb}{{}{4}{Show estimated r from glmmTMB}{section*.7}{}}
\@writefile{toc}{\contentsline {subsubsection}{Show confidence intervals from glmmTMB}{4}{section*.8}\protected@file@percent }
\newlabel{show-confidence-intervals-from-glmmtmb}{{}{4}{Show confidence intervals from glmmTMB}{section*.8}{}}
\gdef \@abspage@last{4}
