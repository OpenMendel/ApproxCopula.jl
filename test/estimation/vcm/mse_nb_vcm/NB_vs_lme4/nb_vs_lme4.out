\BOOKMARK [2][-]{section*.1}{\376\377\000C\000o\000m\000p\000a\000r\000i\000n\000g\000\040\000P\000a\000r\000a\000m\000e\000t\000e\000r\000\040\000E\000s\000t\000i\000m\000a\000t\000e\000s}{}% 1
\BOOKMARK [2][-]{section*.2}{\376\377\000C\000o\000m\000p\000a\000r\000i\000n\000g\000\040\000F\000i\000t\000\040\000T\000i\000m\000e\000s\000\040\000\050\000C\000I\000\040\000t\000i\000m\000e\000s\000\051\000\040\000i\000n\000\040\000S\000e\000c\000o\000n\000d\000s}{}% 2
\BOOKMARK [1][-]{section*.3}{\376\377\000l\000m\000e\0004\000:\000\040\000g\000l\000m\000e\000r\000.\000n\000b}{}% 3
\BOOKMARK [2][-]{section*.4}{\376\377\000S\000h\000o\000w\000\040\000e\000s\000t\000i\000m\000a\000t\000e\000d\000\040\000r\000\040\000f\000r\000o\000m\000\040\000l\000m\000e\0004\000:\000\040\000g\000l\000m\000e\000r\000.\000n\000b}{section*.3}% 4
\BOOKMARK [3][-]{section*.5}{\376\377\000S\000h\000o\000w\000\040\000c\000o\000n\000f\000i\000d\000e\000n\000c\000e\000\040\000i\000n\000t\000e\000r\000v\000a\000l\000s\000\040\000f\000r\000o\000m\000\040\000l\000m\000e\0004\000:\000\040\000g\000l\000m\000e\000r\000.\000n\000b}{section*.4}% 5
\BOOKMARK [1][-]{section*.6}{\376\377\000g\000l\000m\000m\000T\000M\000B\000:\000\040\000g\000l\000m\000m\000T\000M\000B}{}% 6
\BOOKMARK [2][-]{section*.7}{\376\377\000S\000h\000o\000w\000\040\000e\000s\000t\000i\000m\000a\000t\000e\000d\000\040\000r\000\040\000f\000r\000o\000m\000\040\000g\000l\000m\000m\000T\000M\000B}{section*.6}% 7
\BOOKMARK [3][-]{section*.8}{\376\377\000S\000h\000o\000w\000\040\000c\000o\000n\000f\000i\000d\000e\000n\000c\000e\000\040\000i\000n\000t\000e\000r\000v\000a\000l\000s\000\040\000f\000r\000o\000m\000\040\000g\000l\000m\000m\000T\000M\000B}{section*.7}% 8
